index.html

```diff
@@ -3,7 +3,7 @@
         <title>My website</title>
```

Comment:
I notice that this is an html file rather than php, which means you might still have that caching issue we talked about earlier today. Let me know if you didn't catch what I was saying then and we can go over it.

Speaking of which, I know you asked if there was some kind of written documentation of what I cover. Sorry I don't have anything like that yet (I may put it together for future years, but I'm winging it for now) but you might want to figure out a way to take quick notes. There are many tools that might help such as Evernote, Dropbox Paper (my personal favorite), or Trello (which you've already signed up for). Whatever you use, I think it could be helpful to have a list of notes somewhere. I know we're moving through things quickly and it can be easy to have stuff slip through the cracks. Having said that, I think things are going to slow down a bit next week once everyone starts branching out and working on their own things.